consensus/clique/api.go

```diff
@@ -117,3 +119,60 @@ func (api *API) Discard(address common.Address) {
 
 	delete(api.clique.proposals, address)
 }
+
+type CliqueStatus struct {
```

Comment:
There's no need to prefix this with `Clique`. You're already in the `clique` package, it would be stuttering `clique.CliqueStatus`.

Reply:
Also please add type docs.